ppr-ui/src/components/dialogs/StaffPaymentDialog.vue

```diff
@@ -5,14 +5,14 @@
     @proceed="proceed($event)"
   >
     <template v-slot:content>
-      <staff-payment-component
-        :staffPaymentData="staffPaymentData"
-        :validate="validating"
-        :displaySideLabel="false"
-        :displayPriorityCheckbox="false"
-        @update:staffPaymentData="onStaffPaymentDataUpdate($event)"
-        @valid="valid = $event"
-      />
+<!--      <StaffPayment-->
```

Comment:
Where is the staff payment now?

Reply:
We have a localized version of it. That being said, i'll probably re-write the shared component in vue3 and port it back. 
As of right now, this is a pending to-do item!